support-frontend/app/views/newContributions.scala.html

```diff
@@ -39,8 +39,8 @@
         Canada (CA$)</a></li></ul></details></header><main role="main" class="gu-content__main">
       <div class="gu-content__content gu-content__content-contributions gu-content__content--flex">
         <div class="gu-content__blurb">
-          <h1 class="header ">Help us deliver the independent journalism the world needs</h1>
-          <p class="blurb">
+          <h1 class="gu-content__blurb--header">Help us deliver the independent journalism the world needs</h1>
```

Comment:
this is to keep it in line with the actual template. We **need** to get a better solution soon!